initramfs/scripts/local-bottom/fixrtc-mount

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+# initramfs local-bottom script for fixrtc-mount
+
+PREREQ=""
+
+# Output pre-requisites
+prereqs()
+{
+        echo "$PREREQ"
+}
+
+case "$1" in
+    prereqs)
+        prereqs
+        exit 0
+        ;;
+esac
+
+# This script looks at root file system files to get an accurate as possible
+# date. It runs when we already have a mounted root, as opposed to the 'fixrtc'
+# script, which uses the last mount time of the root file system. In devices
+# where there is no battery for the RTC, 'fixrtc' is not effective, as the mount
+# time will always be inaccurate. This script helps in those cases.
+
+BROKEN_CLOCK=""
+
+# shellcheck disable=SC2013
```

Comment:
IMO this should be unconditional because this makes integration easier (fixrtc is another fact we'd have to convey to gadget makers) and this simplifies testing (one less moving part)